src/code/z_face_reaction.c

```diff
@@ -1,3 +1,176 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_face_reaction/func_800F1250.s")
+/**
+ * Indexes of this array:
```

Comment:
```suggestion
 * Indices of this array:
```
English grammar sucks with all its exceptions, but indices is the proper plural of index.

Reply:
Man, my reviewing game's really off today if I missed that...